libcontainer/configs/validate/validator.go

```diff
@@ -253,16 +253,72 @@ func cgroupsCheck(config *configs.Config) error {
 	return nil
 }
 
+func checkIDMapMounts(config *configs.Config, m *configs.Mount) error {
+	if !m.IsIDMapped() {
+		return nil
+	}
+
+	if !m.IsBind() {
+		return fmt.Errorf("invalid mount %+v: GidMappings/UidMappings is supported only for mounts with the option 'bind'", m)
```

Comment:
Overall, I don't like printing out the whole `struct Mount` in an error message.

Can a mount be uniquely identified by its destination?

Same for similar errors below.

Reply:
Makes sense. Sorry for asking the same q over and over :)